AWS_FOR_FLUENT_BIT_STABLE_VERSION

```diff
@@ -1 +1 @@
-2.21.2
\ No newline at end of file
+2.21.3
```

Comment:
sorry I missed this
we can;t bump stalbe until the new release is out
this will cause the automation to fial